lib/PhasarLLVM/TaintConfig/TaintConfig.cpp

```diff
@@ -106,13 +106,13 @@ findAllFunctionDefs(const ProjectIRDB &IRDB, llvm::StringRef Name) {
       FnDefs.push_back(F);
     }
   } else if (FnDefs.size() > 1) {
-    std::cerr << "The function name '" << Name.str()
-              << "' is ambiguous. Possible candidates are:\n";
+    llvm::errs() << "The function name '" << Name.str()
```

Comment:
```suggestion
    llvm::errs() << "The function name '" << Name
```

llvm::raw_ostream supports direct output of llvm::StringRef without converting to std::string;

same below

Reply:
done, also elsewhere.